content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -76,9 +76,20 @@ is the name of a remote service to which a connection is made. It corresponds to
 the [Resource](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service)
 for the local service.
 
-| System property / Environment variable | Description |
-| -------------------------------------- | ----------- |
-| `otel.instrumentation.common.peer-service-mapping` / `OTEL_INSTRUMENTATION_COMMON_PEER_SERVICE_MAPPING` | Used to specify a mapping from host names or IP addresses to peer services, as a comma-separated list of `<host_or_ip>=<user_assigned_name>` pairs. The peer service is added as an attribute to a span whose host or IP address match the mapping. For example, if set to `1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api`, requests to `1.2.3.4` will have a `peer.service` attribute of `cats-service` and requests to `dogs-abcdef123.serverlessapis.com` will have an attribute of `dogs-api`.
+{{% config_option name="otel.instrumentation.common.peer-service-mapping" %}}
+  Used to specify a mapping from host names or IP addresses to peer services, as a
+  comma-separated list of `<host_or_ip>=<user_assigned_name>` pairs. The peer
+  service is added as an attribute to a span whose host or IP address match the
+  mapping.
+
+  For example, if set to the following:
+
+    1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api
+
+  Then, requests to `1.2.3.4` will have a `peer.service` attribute of
+  `cats-service` and requests to `dogs-abcdef123.serverlessapis.com` will have
+  an attribute of `dogs-api`.
```

Comment:
Notice how we can make nicer presentation of the **Description** now (vs. when it was in a table cell).